src/Components/Modal/GameInfo.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {Button, Grid, TextField} from '@material-ui/core';
+import { render } from '@testing-library/react';
+import Cookies from 'universal-cookie';
+
+
+class GameInfo extends React.Component {
+    constructor(props) {
+        super(props);
+        this.customNodeRef = React.createRef();
+        //let hyparlink = props.hyparlink || new Hyparlink();
+        this.state = {level:'Hard', playerList:'ID1', gameDS:'AVL'};
+        this.handleInput = this.handleInput.bind(this);
+        this.submitDSG = this.handleInput.bind(this)
+    }
+
+
+    handleInput = (e) => {
+        this.setState({ [e.target.name]: e.target.value });
+        console.log(this.state)
+
+        const cookies = new Cookies();
+        cookies.set('level', this.state.level, { path: '/' });
+        cookies.set('playerList', this.state.playerList, { path: '/' });
+        cookies.set('gameDS', this.state.gameDS, { path: '/' });
+    };
+
+
+
+render(){
+
+    return(
+        <form>
+        <Grid container spacing={1} style={{ marginBottom: '5em' }}>
+            <Grid item sm={6} md={6} lg={6}>
+
+                <select value={this.state.level} onInput={this.handleInput} name='level' label='Difficulty Level' style={{ marginBottom: '1em' }}  >
```

Comment:
Please create an issue to fix the bug that it having input from the previous choice. The reason could be ```onInput``` because the bug was not there when it was ```onChange()```.